applications/dashboard/models/class.usermodel.php

```diff
@@ -3454,75 +3454,113 @@ public function PasswordReset($UserID, $Password) {
     * @param array $User
     * @param boolean $PasswordOK
     */
-   public static function RateLimit($User, $PasswordOK) {
-      if (!Gdn::Cache()->ActiveEnabled()) return FALSE;
-//      $CoolingDown = FALSE;
+    public static function RateLimit($User, $PasswordOK) {
+        if (function_exists('apc_store')) {
```

Comment:
The check to `Gdn::Cache()->ActiveEnabled()` must come first. On multiple application server environments you need a global check.